netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -56,6 +68,17 @@
      * Base.
      */
     private final transient Base base;
+    
+    /**
+     * Last cache clearance time.
+     * */
+     private transient Date lastChecked = new Date();
```

Comment:
@prahladyeri we don't need this variable if you will schedule the cleanup as above